torch/jit/_recursive.py

```diff
@@ -80,6 +80,10 @@ def copy_to_script_module(original, stubs):
     # Copy links to Python methods so they can be resolved when compiling
     for name in dir(original):
         item = getattr(original, name)
+        if hasattr(script_module, name):
```

Comment:
Will this prevent redefining parent attributes ?

Reply:
There's no parent/child link between the classes, everything gets copied over and after that each one can be modified independently of the other. This check is to prevent re-assigning things like Python builtins and existing methods

Reply:
Maybe add comment